include/zephyr/bluetooth/mesh/cfg_cli.h

```diff
@@ -1644,6 +1644,17 @@ struct bt_mesh_comp_p1_model_item *bt_mesh_comp_p1_item_pull(
 struct bt_mesh_comp_p1_ext_item *bt_mesh_comp_p1_pull_ext_item(
 	struct bt_mesh_comp_p1_model_item *item, struct bt_mesh_comp_p1_ext_item *ext_item);
 
+/** @brief Unpack a list of key index entries from a buffer.
+ *
+ * @param buf Message buffer.
```

Comment:
```suggestion
/** @brief Unpack a list of key index entries from a buffer.
 *
 * On success, @c dst_cnt is set to the amount of unpacked key index entries.
 *
 * @param buf Message buffer containing encoded AppKey or NetKey Indexes.
```

Reply:
This part is missing: `@param buf Message buffer containing encoded AppKey or NetKey Indexes.`

Reply:
Sry, missed it